MilitaryManager.Core/DTO/Attachments/UploadPdfSignedDTO.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace MilitaryManager.Core.DTO.Attachments
+{
+    public class UploadPdfSignedDTO
+    {
+        public int Id { get; set; }
```

Comment:
Could you remove this DTO and use an id parameter instead?

Reply:
No, as base.service.ts PUT method accepts (record: TModel), not (id: number)